### ISSUE TITLE
project website?

### ISSUE THREAD

**Issue by jessicajonesunitedkingdom** [emoon/HippoPlayer#103]
hi

maybe you can make a small project website with link to windows version to download?
Your player is really unique, since it's open source, cross platform and support A GIANT amount of oldsk00l music!

So I think a small website on github can attract more users, so more bug reports etc

I olny accidently found your player when googling 'adplug' .
Currently it's stays away off the radar...

**Comment by jessicajonesunitedkingdom** [comment]
something like this: http://adplug.github.io/

https://pages.github.com/

**Comment by emoon** [comment]
Yeah. The plan is to have a website for it. The reason it hasn't been done yet is that I want HippoPlayer to fly under the radar for a bit longer until it's more complete.
